src/successModal/index.tsx

```diff
@@ -1,100 +1,84 @@
 import styled from 'styled-components'
-import { color, font, space, modalSize } from '_utils/branding'
 import SuccessModal from './SuccessModal'
 
-const footerHeight = '96px' /* = padding + content */
+import { color, space, responsiveBreakpoints } from '_utils/branding'
 
-const StyledSuccessModal = styled(SuccessModal)`
-  & .kirk-successModal {
-    background-color: ${color.successBackground};
-    color: ${color.white};
-  }
+import IllustratedSection from 'layout/section/illustratedSection'
 
-  & .kirk-modal--large {
-    text-align: center;
-  }
+import Modal from 'modal'
+import Button from 'button'
+import TextDisplay1 from 'typography/display1'
 
-  & .kirk-modal.kirk-successModal .kirk-modal-dialog {
+const StyledSuccessModal = styled(Modal)`
+  padding: 0;
+  text-align: center;
+  background-color: ${color.successBackground};
+
+  .kirk-modal-dialog {
+    display: flex;
+    justify-content: center;
+    padding: 0;
     margin: 0 auto;
-    max-width: ${modalSize.m};
-    height: 100%;
-    padding: ${space.xl} 0 ${footerHeight} 0;
+    width: 100%;
+    border-radius: 0;
     background-color: ${color.successBackground};
   }
 
-  /* Stronger selector to override Modal component styling */
-  &.kirk-modal-dimmer .kirk-modal.kirk-successModal .kirk-modal-dialog.kirk-modal-dialog {
-    width: auto;
-  }
-
-  & .kirk-modal--large.kirk-successModal .kirk-modal-dialog {
-    padding: ${space.xl} 0;
-  }
-
-  & .kirk-modal.kirk-successModal .kirk-modal-body {
-    padding: ${space.xl};
-    font-size: ${font.l.size};
-    line-height: ${font.l.lineHeight};
-    text-align: center;
-    align-items: center;
-  }
-
-  & .kirk-modal--large.kirk-successModal .kirk-modal-body {
-    position: relative;
+  .kirk-modal-body {
     display: flex;
-  }
-
-  & .kirk-modal--large .kirk-successModal-bodyItem {
-    width: 50%;
-  }
+    min-height: 100vh;
```

Comment:
Where you able to test it on different mobile devices? We had a lot of issue in the past with `100vh` as it's not always computed the same way on Android of iOS (for instance when the browser address bar disappear).

Reply:
Thanks for the feedback, I'm going test it

Reply:
actually I've found that `min-height: -webkit-fill-available;` would resolve this issue

Reply:
I don't understand your last comment; min-height: 100vh is still used in the last snapshot.